src/core_ocean/shared/mpas_ocn_tidal_potential_forcing.F

```diff
@@ -0,0 +1,663 @@
+          call mpas_log_write('  NodalPhase $r', realArgs=(/ tidalConstituentNodalPhase(iCon) /))
+          call mpas_log_write('  Type $i', intArgs=(/ tidalConstituentType(iCon) /))
+          call mpas_log_write(' ')
+        end do
+
+        block_ptr => block_ptr % next
+      end do      
+
+
+   end subroutine ocn_tidal_potential_forcing_init!}}}
+
+
+!***********************************************************************
+!
+!  routine tidal_constituent_factors
+!
+!> \brief   Initialized tidal constituent factors 
+!> \author  Steven Brus
+!> \date    September 2019
+!> \details
+!>  This routine initializes the ampiltude, frequency, love numbers, 
+!>  astronomical argument and nodal factors for each tidal constituent
+!>  Nodal factor equations are from:
+!>         "Manual of Harmonic Analysis and Prediction of Tides"
+!>          by Paul Schureman, Special Publication #98, US Coast
+!>          and Geodetic Survey, Department of Commerce (1958).
+!
+!-----------------------------------------------------------------------
+
+   subroutine tidal_constituent_factors(constituentList,nTidalConstituents,refTime, &
+                                     tidalConstituentFreqency, &
+                                     tidalConstituentAmplitude, &
+                                     tidalConstituentLoveNumbers, &
+                                     tidalConstituentNodalAmplitude, &
+                                     tidalConstituentAstronomical, &
+                                     tidalConstituentNodalPhase, &
+                                     tidalConstituentType, &
+                                     iErr) !{{{
+
+      type(char_array), dimension(:), intent(in) :: constituentList
+      integer, intent(in) :: nTidalConstituents
+      type (MPAS_Time_Type), intent(in) :: refTime
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentAmplitude
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentFreqency
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentLoveNumbers
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentNodalAmplitude
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentAstronomical
+      real (kind=RKIND), dimension(:), intent(out) :: tidalConstituentNodalPhase
+      integer, dimension(:), intent(out) :: tidalConstituentType
+      integer, intent(out) :: iErr
+
+      integer :: j
+      integer :: refDay
+      integer :: refYear
+      integer :: refHour
+      !integer :: inty,time_mjd
+      !real (kind=RKIND) :: T,s,h,p,N
+      real (kind=RKIND) :: deg2rad
+      real (kind=RKIND) :: s,p,h,p1,n,I,nu,xi,nup,nup2,pc,T
+
+      iErr = 0
+
+      call mpas_get_time(refTime, YYYY=refYear,DoY=refDay,H=refHour)
+      refDay = refDay + 1
+
+      ! Calculate locations of sun and moon
+      call orbit(refYear,refDay,refHour, &
+                 s,p,h,p1,N,I,nu,xi,nup,nup2,pc)
+
+
+      !! From tide design doc
```

Comment:
Do we need to have these comments currently if we link to the tide design doc in the header?

Reply:
Do you think we should keep them in there? I'm fine either way. I can also put in a more specific comment.

Reply:
Let's keep them for now, I don't think this will hurt and it is more vs less information.  We don't compile it so minimally it will help jog memories or make connections to the design doc.